.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Docs: https://github.com/marketplace/actions/close-stale-issues
+#
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is being marked stale because it has been open for 60 days with no activity. Please comment if this issue is still affecting you. If there is no change, this issue will be closed in 30 days.'
+          stale-pr-message: 'This PR is being marked stale because it has been open for 80 days with no activity. Please upadte the PR or ask for a fresh review'
```

Comment:
```suggestion
          stale-pr-message: 'This PR is being marked stale because it has been open for 60 days with no activity. Please update the PR or ask for a fresh review'
```